### PR TITLE
USWDS-Site - Home: Update showcase

### PR DESCRIPTION
**Pull request by bonnieAcameron**
# Summary

1. Updated showcase
2. Added NIH Library file as webp
3. added NIH preview 
4. Moved that preview to slot 1
5. Removed the Plain language site preview

## Related issue

Closes #_[2087](https://github.com/uswds/uswds-site/issues/2087)_

## Preview link

[Preview Showcase](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/bc-update-showcase-nih/)

## Testing and review

1. Please review that I have uploaded the NIH.webp file in the appropriate location
2. Please ensure that the screenshot is of good quality 
3. Make sure that the grid still renders correctly

### CODE REVIEW

File: pages/home.md [hunk 1]
```diff
@@ -58,6 +58,10 @@ graphic_list:
   <div class="grid-container">
     <h2 class="font-lang-xl margin-y-0 line-height-serif-2">Showcase</h2>
     <div class="grid-row grid-gap tablet:margin-top-3">
+      <div class="tablet:grid-col">
+        <img src="{{ site.baseurl }}/img/webp/home/showcase/NIH.webp" alt="Preview image of National Institutes of Health Library" width="960" height="540">
```
Comment (amyleadem) [inline pages/home.md:62]: To match the other images, the alt text should mention that it is an image of the website.
```suggestion
        <img src="{{ site.baseurl }}/img/webp/home/showcase/NIH.webp" alt="Preview image of National Institutes of Health Library home page" width="960" height="540">
```

**Review by amyleadem — CHANGES_REQUESTED** [review]
@bonnieAcameron I added some notes related to the image size and location of the markup. Let me know if I can help at all with these edits. Other than that, looking good!

File: img/webp/home/showcase/NIH.webp
[the remark this answers is not in the document]
Comment (bonnieAcameron) [reply img/webp/home/showcase/NIH.webp:1]: removed and uploaded with one less than 50kb

File: pages/home.md [hunk 2]
```diff
@@ -83,10 +87,6 @@ graphic_list:
         <img src="{{ site.baseurl }}/img/webp/home/showcase/seedfund.webp" alt="Preview image of Seed Fund home page" width="960" height="540">
         <a href="https://seedfund.nsf.gov/">America’s Seed Fund</a>
       </div>
-      <div class="tablet:grid-col">
-        <img src="{{ site.baseurl }}/img/webp/home/showcase/plainlanguage.webp" alt="Preview image of Plain Language home page" width="960" height="540">
-        <a href="https://plainlanguage.gov/">Plain Language</a>
-      </div>
```
[the remark this answers is not in the document]
Comment (bonnieAcameron) [reply pages/home.md]: Moved vote.gov to line 2

**Comment by bonnieAcameron** [comment]
Hi @amyleadem ! I made all of the requested changes. Can you re-review for me? Thank you!

**Review by amyleadem — APPROVED** [review]
This looks good to me! Thanks, @bonnieAcameron. I'll pass it to @mahoneycm to get a second set of eyes on this and then we can pass to @thisisdano.

**Review by mahoneycm — APPROVED** [review]
Looks great! Thanks Bonnie

- [x] Checked multiple screensizes
- [x] Styles inline with other homepage previews
- [x] Expected screenreader output met

**Comment by mahoneycm** [comment]
@mejiaj Could this be a quick merge when you have a sec?

**Review by mejiaj — APPROVED** [review]
Looks good;  we can merge once checks pass.

**Comment by mahoneycm** [comment]
@mejiaj Bumping since checks passed 👍

**Pull request merged**